src/openslide-decode-tiff.c

```diff
@@ -299,6 +299,17 @@ bool _openslide_tiff_read_tile(struct _openslide_tiff_level *tiffl,
       return false;
     }
 
+    // In some Aperio images, one tile (typically 0, 0), in one or more levels,
+    // is partly overwritten with some unknown data. This seems to happen
+    // from byte 0 or byte 1 of the tile data. Such garbled tiles are
+    // impossible to read, so just return false!
+    // The other half of this fix/hack consists of calling render_missing_tile
+    // from decode_tile (in openslide-vendor-aperio) if _openslide_tiff_read_tile
+    // returns false.
+    const unsigned char *bytes = buf;
+    if (bytes[0] == 0x11 || (bytes[0] == 0xff && bytes[1] == 0x11))
```

Comment:
This shouldn't be in driver-independent code.  Instead, I'd suggest that the Aperio `decode_tile` handle the error from `_openslide_tiff_read_tile` by calling `_openslide_tiff_read_tile_data` and checking for the bad data itself.  Since this will rarely happen, the performance penalty should be acceptable.